packages/theme/src/abstracts/_variables-bootstrap.scss

```diff
@@ -11,25 +11,24 @@ $white: #fff;
 $black: #000;
 
 // Gray
-$gray-base: #576373;
+$gray-base: #5c5c5c;
+$gray-bg: #f2f7fa;
```

Comment:
I'm unsure on the naming for this `$gray-bg` variable. It may be better just to map `$body-bg` to this hex directly if that is the only place it is being used.

Reply:
It may be better to describe this value as `$bg-<modifier>` so it has more of a semantic meaning rather than a color association. Ex: a future brand may use a light, but non-gray color for this value.